client/state/admin-menu/test/reducer.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import deepFreeze from 'deep-freeze';
```

Comment:
Can we avoid using deepFreeze here? It seems that it is not needed. I may be mistaken though!

Reply:
It's a pattern in Calypso to deepFreeze the state in a reducer test in order to check that the reducer doesn't _mutate_ the state.

Reply:
Thanks for clarifying it!